tests/foreman/sys/test_fam.py

```diff
@@ -54,10 +54,20 @@ def install_import_ansible_role(module_target_sat):
 
 
 @pytest.fixture(scope='module')
-def setup_fam(module_target_sat, module_sca_manifest, install_import_ansible_role):
+def setup_fam(
+    module_target_sat, module_sca_manifest, install_import_ansible_role, module_capsule_configured
+):
     # Execute AAP WF for FAM setup
     Broker().execute(workflow='fam-test-setup', source_vm=module_target_sat.name)
 
+    # Edit settings so FAM playbooks use the capsule we setup and then
+    # delete it from the Satellite so that the tests can use it.
+    settings.fam.server.foreman_proxy = module_capsule_configured.hostname
```

Comment:
I don't think it's good idea to overwrite settings this way. It seems settings are session scoped, so they will be broken for further test-cases.

Reply:
We need a way for the settings file to point to our capsule which was set up during test execution. Unless we have a long running capsule host that doesn't get torn down, then I think we have to do it this way. Plus, the capsule hostname should change between each session along with the settings.